scripts/load_print_styles_from_cdn.sh

```diff
@@ -8,7 +8,7 @@ if [ -z ${1+x} ]; then
   exit 1
 fi
 
-CDN_URL=${CDN_URL:-"https://cdn.eq.gcp.onsdigital.uk"}
+CDN_URL=${CDN_URL:-"https://cdn.ons.gov.uk"}
```

Comment:
This highlights that we weren't passing a CDN_URL and in formal envs, we were building using dev cdn.

Reply:
Didn't get this , can you elaborate on this please